sound/soc/sof/intel/hda-dsp.c

```diff
@@ -405,6 +405,26 @@ static int hda_dsp_update_d0i3c_register(struct snd_sof_dev *sdev, u8 value)
 	return 0;
 }
 
+static bool hda_dsp_d0i3_streaming(struct snd_sof_dev *sdev)
+{
+	struct snd_pcm_substream *substream;
+	struct snd_sof_pcm *spcm;
+	int dir;
+
+	list_for_each_entry(spcm, &sdev->pcm_list, list) {
+		for_each_pcm_streams(dir) {
+			substream = spcm->stream[dir].substream;
+			if (!substream || !substream->runtime)
+				continue;
+
+			if (spcm->stream[dir].d0i3_compatible && dir == SNDRV_PCM_STREAM_PLAYBACK)
```

Comment:
This function is for playback only. Should we add "playback" to the function name? For example, hda_dsp_d0i3_playback_streaming.

Reply:
or `hda_dsp_d0i3_playback_active()`?

Reply:
sure thanks